File: translation_output/magicoder/codenet/java/python-sanitized/p03250.py
# Python code

a = int(input())
b = int(input())
c = int(input())

if a >= b and a >= c:
    if b >= c:
        print(int(str(a) + str(b)) + c)
    else:
        print(int(str(a) + str(c)) + b)
elif b >= a and b >= c:
    if a >= c:
        print(int(str(b) + str(a)) + c)
    else:
        print(int(str(b) + str(c)) + a)
else:
    if a >= b:
        print(int(str(c) + str(a)) + b)
    else:
        print(int(str(c) + str(b)) + a)

File: Results/DER/Translation/translation_output/misleading_test/granite/codenet/python/java/p03250.java
System:
You are an expert java programmer and assistant

Question:
Translate the following code from python to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
1 5 2

Expected output:
-2

```
abc = list(map(int, input().split()))
abc.sort()
print(int(str(abc[2]) + str(abc[1])) + abc[0])
```


Answer:
```java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int[] abc = new int[input.length];
        for (int i = 0; i < input.length; i++) {
            abc[i] = Integer.parseInt(input[i]);
        }
        Arrays.sort(abc);
        int result = Integer.parseInt(String.valueOf(abc[2]) + String.valueOf(abc[1])) + abc[0];
        System.out.println(result);
    }
}
```

Test input:
1 5 2

Expected output:
-2<|endoftext|>
